src/lib/components/ManagePanel.js

```diff
@@ -86,7 +86,8 @@ export default class ManagePanel extends React.Component {
 
   getDate(time) {
     if (time === NEXT_OPEN) {
-      return browser.i18n.getMessage('manageDateNext');
+      const message = browser.i18n.getMessage('manageDateNextOpen');
+      return message.split('<br>', '2')[0] || message;
```

Comment:
Yeah… At least it "breaks" gracefully, and it seems less scary than injecting raw HTML (it's from a safe-ish source, but still…). And, if you change your mind about that, you can still use the same string.